### DISCUSSION TITLE
Capitation Arrangement With Underlying Fee Schedule

### DISCUSSION THREAD

**Discussion by angelamarie0975** [CMSgov/price-transparency-guide#483] [Q&A]
How exactly should the JSON file be laid out when the provider is capitated and we need to also display the underlying fee schedule per procedure. In the capitation example, the layout flow is plan/network arrangement (Capitation)/negotiated price/covered services. How do we apply both the capitated agreement and the underlying fee schedule under the In-Network Object for a capitated negotiated agreement? Would we we have 2 In-Network Objects for the same providers - one listed as Capitation negotiation agreement and the same provider under a FFS negotiation agreement for the underlying fee schedule for the same procedures listed in the Covered Services Object in the Capitation agreement?

Blurb from white papers = The “Underlying Fee-Schedule Rate,” which is the rate a Plan or Issuer uses to determine a Member’s cost-sharing liability for an Item or Service, if the rate is different from the Negotiated Rate or Derived Amount.  45 C.F.R. § 147.210(a)(2)(xxii).  Thus, for example, a Plan or Issuer that pays an InNetwork Provider a capitated rate might use a fee schedule to determine the amount on which Member coinsurance should be based.  A fee schedule for a Member’s 30% coinsurance might value an office visit at $100, in which case the Member pays $30 coinsurance.  **_In this example, the Plan or Issuer would report the capitated rate as the Negotiated Rate and the Underlying Fee Schedule Rate as $100._**

**Comment by angelamarie0975** [comment]
Also - one more question - assuming that the examples are laid out differently in Capitation vs the other 2 In-Network file examples is because of the nature of Capitation? Why is the Capitation file not following the same schema as indicated on the In-Network Schema?

**Comment by tdfow** [comment]
#26 #21 #10
